ui/pages/institutional/custodian-accounts-connected/custodian-accounts-connected.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import CustodianAccountsConnected from './custodian-accounts-connected';
+
+jest.mock('../../../hooks/useI18nContext', () => ({
+  useI18nContext: jest.fn(),
+}));
+
+describe('CustodianAccountsConnected', () => {
+  const useI18nContextMock = useI18nContext as jest.Mock;
+
+  beforeEach(() => {
+    useI18nContextMock.mockReturnValue((key: string) => key);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render CustodianAccountsConnected', async () => {
+    const { container, getByText } = render(<CustodianAccountsConnected />);
+    expect(container).toMatchSnapshot();
+
+    await waitFor(() => {
+      expect(
+        getByText('allCustodianAccountsConnectedTitle'),
+      ).toBeInTheDocument();
+    });
```

Comment:
Nit: can we directly assert here?
```suggestion
     expect(
        getByText('allCustodianAccountsConnectedTitle'),
      ).toBeInTheDocument();
```